app/src/main/cpp/src/3rd_party-static/CMakeLists.txt

```diff
@@ -54,6 +50,31 @@ include_directories(
 )
 add_subdirectory(./sqlite_static)
 
+# --- openssl
+set(OPENSSL_INCLUDE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/openssl/include)
+set(OPENSSL_INCLUDE_DIRECTORY ${OPENSSL_INCLUDE_DIRECTORY} PARENT_SCOPE)
+
+set(OPENSSL_LIBS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/openssl/x86_64)
+set(OPENSSL_LIBS_DIRECTORY ${OPENSSL_LIBS_DIRECTORY} PARENT_SCOPE)
+
+include_directories(
+  ${OPENSSL_INCLUDE_DIRECTORY}
+)
+
+add_library(crypto STATIC IMPORTED)
+set_target_properties(crypto
+        PROPERTIES
+        IMPORTED_LOCATION ${OPENSSL_LIBS_DIRECTORY}/libcrypto.a)
+
+add_library(ssl STATIC IMPORTED)
```

Comment:
@LitvinenkoIra we should keep backward compatibility with QNX/Linux build. If this block is Android specific, then please wrap this into `if (ANDROID) .. endif()` block

Reply:
Done in https://github.com/CustomSDL/sdl_core/pull/278/commits/a216eafd54b0faa26281978e71f43be4f02fa3c8